tutorials/plot_compute_covariance.py

```diff
@@ -33,26 +34,40 @@
 # useful if you use resting state as a noise baseline. Here we use the whole
 # empty room recording to compute the noise covariance (tmax=None is the same
 # as the end of the recording, see :func:`mne.compute_raw_covariance`).
+#
+# Keep in mind that you want to match your empty room dataset to your
+# actual MEG data, processing-wise. Ensure that filters
+# are all the same and if you use ICA, apply it ot both your nois and
+# your MEG recordign. In this case we did not filter the data and
+# we don't use ICA. However, we do have bad channels and projections in
+# the MEG data, and, hence, we want to make sure they get stored in the
+# covariance object.
+
+raw_empty_room.info['bads'] = [
+    bb for bb in raw.info['bads'] if 'EEG' not in bb]
+raw_empty_room.add_proj(
+    [pp.copy() for pp in raw.info['projs'] if 'EEG' not in pp['desc']])
```

Comment:
Yes this is true. But as long as it does not, we have to add these lines here. We can open an issue for improving adding projs. Let's keep this PR small.